consensus/hotstuff/committees/threshold_test.go

```diff
@@ -17,3 +18,16 @@ func TestComputeWeightThreshold(t *testing.T) {
 		assert.False(t, boundaryValue < float64(threshold-1))
 	}
 }
+
+// TestComputeTOWeightThreshold tests computing the HotStuff safety threshold
+// for producing a TO.
+func TestComputeTOWeightThreshold(t *testing.T) {
```

Comment:
* Could we rename `i` to `totalWeight`? 
* I would suggest to change the test slightly. There are two equations that must be satisfied for `threshold` to be correct
   * `threshold * 3 > totalWeight`
   * `(threshold - 1) * 3  <= totalWeight`

   The reason why I prefer these checks is that we don't imitate the computation of the threshold.